spec/component/services/oa_workflow_service_spec.rb

```diff
@@ -111,7 +111,7 @@
       it 'calls the publication download job with that file' do
         service.workflow
         expect(PublicationDownloadJob).to have_received(:perform_later).with(activity_insight_oa_file4.id)
-        expect(PublicationDownloadJob).not_to have_received(:perform_later).with(activity_insight_oa_file5.id)
+        expect(PublicationDownloadJob).to have_received(:perform_later).with(activity_insight_oa_file5.id)
```

Comment:
I think we should test at least one of the other files that isn't sent through the job.  Maybe `file6`?